src/igvfd/types/modification.py

```diff
@@ -75,6 +83,22 @@ def summary(self, request, cas, cas_species, modality, fused_domain=None, tagged
 
         return summary
 
+    @calculated_property(schema={
+        'title': 'Samples modified',
+        'description': 'The samples which have been modified with this modification.',
+        'type': 'array',
+        'minItems': 1,
+        'uniqueItems': True,
+        'items': {
+            'title': 'Samples modified',
+            'type': ['string', 'object'],
+            'linkFrom': 'Sample.modifications',
```

Comment:
modifications is on the level of Biosample so it should link from there instead.